### PR TITLE
add known issue for dr secondary lease count quota invalidation

### PR DESCRIPTION
**Pull request by swayne275**

### CODE REVIEW

**Review by vishalnayak — APPROVED** [review]

**Comment by mladlow** [comment]
It might take me a little while to review this but I'd like to request changes, so if you could hold off on merging, that would be helpful.

File: website/content/partials/lease-count-quota-upgrade.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Lease Count Quota Invalidations on DR Secondaries Fixed
+
+Lease count quota invalidation causes DR Secondaries to panic and experience
+a hard shutdown. This issue exists prior to Vault 1.6.6, and 1.7.4. It has been
```
Comment (mladlow) [inline website/content/partials/lease-count-quota-upgrade.mdx:4]:
```suggestion
a hard shutdown. This issue exists prior to Vault 1.6.6, and 1.7.4. It is
```

Comment (mladlow) [reply website/content/partials/lease-count-quota-upgrade.mdx:4]: It won't quite be released when this PR merges, so we can leave the tense ambiguous this way.

File: website/content/docs/upgrading/upgrade-to-1.7.4.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.7.4 - Guides
```
Comment (mladlow) [inline website/content/docs/upgrading/upgrade-to-1.7.4.mdx:3]: Like the 1.6.6 upgrade page, we should put this on the 1.7.0 upgrade page at least, and maybe 1.7.1/1.7.2/1.7.3 (but I'm more on the fence about those).

File: website/data/docs-nav-data.json [hunk 3]
```diff
@@ -1620,10 +1620,18 @@
         "title": "Upgrade to 1.6.3",
         "path": "upgrading/upgrade-to-1.6.3"
       },
+      {
```
Comment (mladlow) [inline website/data/docs-nav-data.json:1623]: If we get rid of the 1.6.6 and 1.7.4 pages, we won't want these links.

**Review by mladlow — CHANGES_REQUESTED** [review]
I think it would be more consistent and more obvious to put the "known issues" on the 1.6.0 and 1.7.0 upgrade pages, as there isn't a known issue with 1.6.6 or 1.7.4. Changing that would mean that you don't need to add a new page for 1.6.6 and 1.7.4.

**Comment by swayne275** [comment]
@mladlow I resolved your comments (thanks!). Let me know if you'd like this info to also be put on the minor 1.6 and 1.7 pages

**Review by mladlow — APPROVED** [review]
I think we don't need to add it to the other pages.

**Pull request merged**